lib/timsort_lib_purecap.c

```diff
@@ -192,8 +193,12 @@ void timSort_unsafe(int *arr)
  */
 void timSort_purecap(int arr[], size_t length)
 {
-	size_t bounds_length = cheri_getlen(arr) / sizeof(int);
+	if (length <= 1)
```

Comment:
Ditto.

Reply:
[Answer to twin question](https://github.com/capablevms/cheri-examples/pull/7#discussion_r515880301)